deployment/templates/deploymentconfig.yaml

```diff
@@ -41,7 +41,11 @@ spec:
         terminationMessagePolicy: File
         volumeMounts:
         - name: config-volume
+          {{- if .Values.development }}
+          mountPath: /var/www/html/config
+          {{- else }}
           mountPath: /opt/app-root/src/config
+          {{- end }}
```

Comment:
I would just like to make sure I understand the need for different mount paths between dev and non-dev here. Would you mind enlighten me @jacobsee?

Reply:
Yeah so this is a weird one... Basically, the difference is that in dev mode, we're using the `httpd` image in a "just `COPY` some files into the `DocumentRoot`" sort of way[1]... but in production, we're doing a _real_ s2i to build that image. Part of the s2i build script in the `httpd` image changes the `DocumentRoot` when it is being used in that way[2]. If we could redo the dev side to also use s2i, that would be ideal, but I don't have any quick thoughts on doing that since the app build must happen externally in a container with NPM.

[1] https://github.com/rht-labs/open-management-portal-frontend/blob/c28b1dc0a5f0ca11c9987fcdfdf8f485980c89c8/development/templates/buildconfig.yaml#L27
[2] https://github.com/sclorg/httpd-container/blob/cd65777b68451c6725a3931caee21d195a2b3c7c/2.4/root/usr/share/container-scripts/httpd/common.sh#L91

Reply:
Seems like a good enough work around and since Petter's ok with merging otherwise. Shall we merge?

Reply:
I'd like to, yes!